Dockerfile.windows

```diff
@@ -19,7 +19,8 @@ SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPref
 RUN Write-Host ('Creating folders'); `
     New-Item -Type Directory -Path /local; `
     New-Item -Type Directory -Path /out/bin; `
-    New-Item -Type Directory -Path /out/config;
+    New-Item -Type Directory -Path /out/config; `
+    New-Item -Type Directory -Path /work/bin;
```

Comment:
I thought you said you were going to be able to build the package from /out?

Reply:
In that case, maybe just put the output files at /work/out? Then you can have googet pull things from underneath /work, without having two copies of everything and without polluting /work with random build products.